inputs/org.osbuild.containers

```diff
@@ -36,6 +36,21 @@ SCHEMA = r"""
       }
     }
   },
+  "source-storage-options": {
+    "type": "object",
+    "required": ["name"],
+    "additionalProperties": false,
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "The name to use for the image"
+      },
+      "storage-path": {
```

Comment:
This property gets added by `osbuild/sources: add containers-storage`, but is unused until the last commit which actually renames this property. Not a big deal, but it makes the history hard to read.

Reply:
Oh yeah sorry I changed this later. I will fix this up so the commit history makes sense